src/testdir/test_xxd.vim

```diff
@@ -415,4 +415,132 @@ func Test_xxd_little_endian_with_cols()
   bwipe!
 endfunc
 
+func Test_xxd_color()
```

Comment:
The test should use vim's screendump features.

Reply:
I don't know anything about vim's screendump features.

Reply:
`:h terminal-screendump`

Reply:
I have never used vim.